src/components/popup/Popup.tsx

```diff
@@ -60,7 +60,7 @@ export const Popup = React.forwardRef<HTMLDivElement, PopupProps>(
       <PopupContext.Provider value={{ onHide }}>
         {show &&
           ReactDOM.createPortal(
-            <FocusLock {...props} returnFocus>
+            <FocusLock returnFocus autoFocus={false} {...props}>
```

Comment:
@vozbrann what is the benefit to do it?

Reply:
To not focus on the close button after opening

Reply:
The problem was that when opening the popup, the close button appeared focused

Reply:
Hello @vozbrann!

Why do you want to disable by default for all cases this focus (which in my opinion should be there)?
If you need it for your project you can pass it using props, right? 

Also I don't see person who asked you about it, because your description is empty 😬

Reply:
@lazzystep 
In most cases, we have a close button in the header of a modal. So when you open a modal now, to set focus to this button you should press TAB.
Because of the thick indication of focus, I think it's better not to focus on the close button by default.
Check Bootstrap modal https://getbootstrap.com/docs/5.1/components/modal/#live-demo

Reply:
@vozbrann, after discussion with @vlacher12, he purposed that would be awesome if we could make first active element in body be in focus. 

Could check is it possible, please? You can use `AutoFocusInside` component from lib. So idea is to wrap children inside Body.

Reply:
@lazzystep I am not sure if it is expected behavior for modal. chakra-ui, ant.design and bootstrap don't have this feature.
Maybe it's better to implement AutoFocusInside in each project where it's needed?

Reply:
Here's an [example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html) from W3 org. 
Here's an [youtube example](https://www.youtube.com/watch?v=2Q56TbgvN5c) of modal that uses a11y rules

Give it a shot